Client/Application/Delegates/AppDelegate.swift

```diff
@@ -228,7 +228,13 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UIViewControllerRestorati
             icon: UIApplicationShortcutIcon(templateImageName: "quick_action_new_private_tab"),
             userInfo: [:])
         
-        application.shortcutItems = Preferences.Privacy.privateBrowsingOnly.value ? [privateTabItem] : [newTabItem, privateTabItem]
+        let scanQr = UIMutableApplicationShortcutItem(type: "\(Bundle.main.bundleIdentifier ?? "").ScanQR",
```

Comment:
nit: Can you rename it similar other shortcut items ?

`let scanQRCodeItem = `